### PR TITLE
adding header and some UI for all products page

### PR DESCRIPTION
**Pull request by sonia-ko**

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -1,10 +1,36 @@
 import "./App.css";
-
+import Contacts from "./components/header/contacts/ContactsComponent";
```
Comment (Yuliia143) [inline src/App.tsx]: Remove unused import

File: src/App.tsx [hunk 2]
```diff
@@ -1,10 +1,36 @@
...
-    </div>
+    <Router>
+      <div className="App">
+        <Header> </Header>
+        <Switch>
+          <Redirect exact from="/" to="/products" />
+          <Route
+            path="/products"
+            render={() => (
+              <React.Fragment>
```
Comment (Yuliia143) [inline src/App.tsx]: Remove unneeded Fragment

File: src/components/header/productsMenu/ProductsMenu.module.css [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+  text-align: center;
+  font-size: 1em;
+  padding: 0 1vw;
+  -moz-appearance: none;
+  -webkit-appearance: none;
+  font-family: "Prompt", sans-serif;
+  font-weight: 500;
+}
+
+.productsMenu ul li select option {
```
Comment (Yuliia143) [inline src/components/header/productsMenu/ProductsMenu.module.css]: Fix selectors

File: src/components/header/productsMenu/ProductsMenu.tsx [hunk 4]
```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import { productCategories } from "../../../Config";
+import classes from "./ProductsMenu.module.css";
+
+const ProductsMenu: React.FC = () => {
+  console.log(productCategories);
```
Comment (Yuliia143) [inline src/components/header/productsMenu/ProductsMenu.tsx]: Remove all console logs

**Review by Yuliia143 — CHANGES_REQUESTED** [review]

File: src/App.tsx (hunk 1 above)
Comment (sonia-ko) [reply src/App.tsx]: Fixed, thank you.

File: src/components/header/productsMenu/ProductsMenu.module.css (hunk 3 above)
Comment (sonia-ko) [reply src/components/header/productsMenu/ProductsMenu.module.css]: Done, thank you.

File: src/components/header/productsMenu/ProductsMenu.tsx (hunk 4 above)
Comment (sonia-ko) [reply src/components/header/productsMenu/ProductsMenu.tsx]: Removed, thank you

File: src/Root.tsx [hunk 5]
```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import { Provider } from "react-redux";
+import { BrowserRouter as Router } from "react-router-dom";
+import { store } from "./store/store";
+
+type Props = {};
```
Comment (Yuliia143) [inline src/Root.tsx]: Remove empty type

File: src/components/header/HeaderComponent.tsx [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+import Search from "./search/Search";
+import cartIcon from "./shopping-cart.png";
+import userIcon from "./user.png";
+import ProductsMenu from "./productsMenu/ProductsMenu";
+
+const Header: React.FC = () => {
+  return (
+    <div className={classes.header}>
+      <div className={classes.topNavBar}>
+        <Contacts></Contacts>
```
Comment (Yuliia143) [inline src/components/header/HeaderComponent.tsx]: Fix empty tag

File: src/components/header/contacts/ContactsComponent.tsx [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+  const history = useHistory();
+
+  return (
+    <div className={classes.contacts}>
+      <ul className={classes.contactsList}>
+        <li
+          className={classes.contactsItem}
+          onClick={() => history.push("/support")}
+        >
+          Chat with us
```
Comment (Yuliia143) [inline src/components/header/contacts/ContactsComponent.tsx]: Add Link

File: src/components/header/mainMenu/MainMenu.tsx [hunk 8]
```diff
@@ -0,0 +1,27 @@
...
+const MainMenu: React.FC = () => {
+  const history = useHistory();
+  return (
+    <nav className={classes.mainMenuContainer}>
+      <ul className={classes.menuItems}>
+        <li className={classes.menuItem} onClick={() => history.push("/blog")}>
+          Blog
+        </li>
+        <li className={classes.menuItem} onClick={() => history.push("/about")}>
+          About Us
```
Comment (Yuliia143) [inline src/components/header/mainMenu/MainMenu.tsx]: Add Link*2

File: src/Root.tsx (hunk 5 above)
Comment (sonia-ko) [reply src/Root.tsx]: Done, thanks :)

File: src/components/header/HeaderComponent.tsx (hunk 6 above)
Comment (sonia-ko) [reply src/components/header/HeaderComponent.tsx]: Fixed, thank you

File: src/components/header/contacts/ContactsComponent.tsx (hunk 7 above)
Comment (sonia-ko) [reply src/components/header/contacts/ContactsComponent.tsx]: Added, thanks

File: src/components/header/mainMenu/MainMenu.tsx (hunk 8 above)
Comment (sonia-ko) [reply src/components/header/mainMenu/MainMenu.tsx]: Done, thank you

**Review by Yuliia143 — APPROVED** [review]

**Comment by sonia-ko** [comment]
Merged

**Pull request merged**